components/nft-sale-widget/layout/nft-sale/mint-section.tsx

```diff
@@ -250,14 +161,15 @@ const MintSection = ({
                 tokenId: token.tokenID,
                 ipfsHash: activeSale.ipfsHash,
                 whitelist: whitelist[activeSale.ipfsHash] ?? [],
-                balance: balances[token.tokenID] ?? "0"
+                balance: "0"
               }}
               index={parseInt(token.id, 10)}
               isRandom={isRandom}
               isPrivateSale={privateSaleStatus}
               isOriginal={isOriginal}
               loading={loading}
-              refetchBalances={refetchAllBalances}
+              // eslint-disable-next-line @typescript-eslint/no-empty-function
+              refetchBalances={() => {}}
```

Comment:
same as above :)

Reply:
yeah this can be refactored later when I use the context, but the prop can't be removed yet cos we're sharing the same component between the mint and owned section